### ISSUE TITLE
Accessibility: https://ipython.org/install.html

### ISSUE THREAD

**Issue by Meekaylah** [jupyterhub/outreachy#89]
### What page is this for?

[This page](https://ipython.org/install.html) is used to install IPython

### WAVE accessibility report

# Installing [IPython](https://ipython.org/install.html) Page
<img width="348" alt="Screenshot 2022-10-19 at 06 57 08" src="https://user-images.githubusercontent.com/84780375/196608726-aa202b74-cb4b-45a3-bbd1-b11db9798cc1.png">

- The language of this page is not identified.
- This is important because if the language is not identified, users of this site might not be able to read the content of the site in the appropriate language. Also, if there is need for translation, users would not be able to use web browser translation tools because the website does not have an identified language to translate from.
- This also important because users need to be able to understand installation instructions and can't access it in their own language if this is not updated.
- A simple update to the first line of code which is the `html` tag, by adding a `lang` attribute, is needed.

### The HTML element to be changed

```html
This line of code needs to be changed:
<html xmlns="http://www.w3.org/1999/xhtml">
```

### Your proposed HTML (or CSS), after change

```html
This is how the new line of code should look like:
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
```

**Comment by minrk** [comment]
Thank you, we do indeed need to set that attribute

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
